index.bs

```diff
@@ -1726,30 +1732,33 @@ the device's <a>configuration descriptor</a> which is a description of the
 device's capabilities including the interfaces and endpoints it exposes. A class
 can be declared at the device level or for individual interfaces. A device with
 multiple interfaces providing different functions is known as a
-<dfn>composite device</dfn>.
+<strong><em>composite device</em></strong>.
 
 ## Transfers ## {#appendix-transfers}
 
 Whether data is traveling from host to device or the other way around the
-transfer is always initiated by the host. <dfn>OUT transfers</dfn> carry data
-from host to device and may wait until the device acknowledges the data has been
-received. <dfn>IN transfers</dfn> carry data from device to host and may have to
-wait until the device has some data to send. Transfers are executed against one
-of a device's endpoints and there are different kinds depending on what type of
-traffic is being sent.
+transfer is always initiated by the host.
+<strong><em>OUT transfers</em></strong> carry data from host to
+device and may wait until the device acknowledges the data has been received.
+<strong><em>IN transfers</em></strong> carry data from device to
+host and may have to wait until the device has some data to send. Transfers are
+executed against one of a device's endpoints and there are different kinds
+depending on what type of traffic is being sent.
 
   * <dfn>Bulk transfers</dfn> are good for sending lots of data with whatever
     bandwidth is available. This is what is used to read and write data to USB
     mass storage devices.
   * <dfn>Interrupt transfers</dfn> offer guaranteed latency (by reserving
-    bandwidth so that they can't be blocked by a large bulk transfers) but with
-    limited packet sizes. These are used for signaling and for small packets like
-    mouse movements and button presses.
-  * <dfn>Isochronous transfers</dfn> also reserve bandwidth but they don't
-    guarantee delivery. They're used for streaming data like audio and video.
-  * Every device also has a special <dfn>default endpoint</dfn>. While regular
+    bandwidth so that they can't be blocked by a large <a>bulk transfers</a>)
+    but with limited packet sizes. These are used for signaling and for small
+    packets like mouse movements and button presses.
+  * <strong><em>Isochronous transfers</em></strong> also reserve
```

Comment:
We should be able to reference this from the algorithms for isochronous transfers.